bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -606,6 +606,9 @@ impl Client {
         &self,
         action: Option<AccountManagementAction>,
     ) -> Result<Option<String>, ClientError> {
+        if !matches!(self.inner.auth_api(), Some(AuthApi::Oidc(..))) {
+            return Ok(None);
+        }
```

Comment:
```suggestion
        }
        
```

Reply:
You kept some trailing whitespaces, that's pure evil, I admire that.